spec/support/features/vote_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module Features
+  module VoteHelpers
+    def vote_setup (user, post)
+      built_post = user.posts.build({ title: post.title, url: post.url })
+      built_post.save
```

Comment:
Ahh, it only makes sense to do the two things separately if I wanted to check/validate something before saving, right? I guess that doesn't matter in this file because I know that I'm supplying a valid factory.